lib/statsig/application.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Statsig.Application do
+  use Application
+  require Logger
+
+  def start(_type, args) do
+    api_key = Keyword.get(args, :api_key)
+    Application.put_env(:statsig, :api_key, api_key)
```

Comment:
@scohen - do I need to parse out all the keyword options I've configured here and set them?